pages/mi/mi-faq.md

```diff
@@ -204,7 +204,7 @@ It's easy to lose sight of the ‘Big Picture’ when focused on individual task
 - [GitHub & Git Foundations (Playlist)](https://www.youtube.com/watch?list=PLg7s6cbtAD15G8lNyoaYDuKZSKyJrgwB-&v=FyfwLX4HAxM)
 - [Shorter Git/GitHub Tutorial (Playlist)](https://www.youtube.com/watch?v=vR-y_2zWrIE&list=PLWKjhJtqVAbkFiqHnNaxpOPhh9tSWMXIF)
 - [Mastering Markdown (Playlist)](https://www.youtube.com/watch?v=Je5w18nn-e8&list=PLu8EoSxDXHP7v7K5nZSMo9XWidbJ_Bns3)
-- [How to Manually Fix Git Merge Conflicts](https://www.youtube.com/watch?v=g8BRcB9NLp4) - Please note that this video will explain how to fix a merge conflict from the point of view of the repo owner who is trying to merge a pull request. However, it is helpful also when you have to fix a merge conflict on your own local and forked repos.
+- [How to Manually Fix Git Merge Conflicts](https://www.youtube.com/watch?v=g8BRcB9NLp4) - Please note that this video will explain how to fix a merge conflict from the perspective of the repository owner who is trying to merge a pull request. However, it is also helpful when you need to resolve a merge conflict in your own local and forked repositories.
```

Comment:
It seems these changes are related to issue #3502.  

However, the commit history looks problematic:  
- Repository sync appears to have been performed on the `add-Avinash-Codes-profile` branch.  
- Work for issue #3500 was done on the `add-Avinash-Codes-profile` branch.  
- PR #3501 was created from the `add-Avinash-Codes-profile` branch.  
- Issue #3502 was also worked on the `profile` branch.  
- Finally, a new branch `3502-FAQ-Content-for-Clarity-and-Formality` was created from the `add-Avinash-Codes-profile` branch for the pull request.

Please review what happened, close out pr #3501 #3503 and start over carefully by following the steps outlined in [Synchronize Your Repository](https://open-learning-exchange.github.io/#!./pages/mi/mi-github-issues.md#4._Synchronize_Your_Repository) for both issue #3500 and #3502.

Reply:
I am so sorry i forgot to create a different branch for each pull request will fix all the things and raise new pr for both #3500  and #3502